Packs/CheckPoint_FW/Integrations/CheckPoint_FW/CheckPoint_FW.py

```diff
@@ -0,0 +1,833 @@
+
+    outputs = {f'Checkpoint.{endpoint}': printable_result}
+    readable_output = tableToMarkdown(f'CheckPoint data for listing {endpoint}s:', printable_result,
+                                      ['name', 'uid', 'type'], removeNull=True)
+    if testing:
+        return readable_output, outputs, result
+    return_outputs(readable_output, outputs, result)
+
+
+def format_get_object(result: dict, endpoint: str, testing: bool = False):
+    """
+        Formats object info from CheckPoint into Demisto's outputs.
+
+    Args:
+        result (dict): the report from CheckPoint.
+        endpoint (str): The endpoint that needs to be formatted.
+        testing(bool): is this function call is for testing purposes.
+    Returns:
+        str: the markdown to display inside Demisto.
+        dict: the context to return into Demisto.
+        dict: the report from CheckPoint (used for debugging).
+    """
+
+    outputs = {f'checkpoint.{endpoint}': {
+        'name': result.get('name'),
+        'uid': result.get('uid'),
+        'type': result.get('type'),
+        'domain_name': result.get('domain').get('name'),
+        'domain_uid': result.get('domain').get('uid'),
+        'domain_type': result.get('domain').get('type'),
+        'ipv4_address': result.get('ipv4-address'),
+        'ipv6_address': result.get('ipv4-address'),
+        'read_only': result.get('read-only'),
+        'meta_info:creator': result.get('meta-info').get('creator'),
+        'meta_info:last_modifier': result.get('meta-info').get('last-modifier')
+    }}
+    table_data = outputs[f'checkpoint.{endpoint}']
+    readable_output = tableToMarkdown(f'CheckPoint Data for Getting {endpoint}:', table_data,
+                                      removeNull=True)
+    if testing:
+        return readable_output, outputs, result
+    return_outputs(readable_output, outputs, result)
+
+
+def format_add_object(result: dict, endpoint: str, testing: bool = False):
+    """
+        Formats adding object result from CheckPoint into Demisto's outputs.
+
+    Args:
+        result (dict): the report from CheckPoint.
+        endpoint (str): The endpoint that needs to be formatted.
+        testing(bool): is this function call is for testing purposes.
+    Returns:
+        str: the markdown to display inside Demisto.
+        dict: the context to return into Demisto.
+        dict: the report from CheckPoint (used for debugging).
+    """
+    common_output = {
+        'name': result.get('name'),
+        'uid': result.get('uid'),
+        'type': result.get('type'),
+        'domain_name': result.get('domain').get('name'),
+        'domain_uid': result.get('domain').get('uid'),
+        'domain_type': result.get('domain').get('type'),
+        'meta_info:creator': result.get('meta-info').get('creator'),
+        'meta_info:last_modifier': result.get('meta-info').get('last-modifier')
+    }
+    unique_outputs = {}
+    if endpoint == 'application-site':
+        unique_outputs = {
```

Comment:
why those fields are not returned in `get` commands?

Reply:
Can you please explain what you mean?